rewrite-java-17/src/main/java/org/openrewrite/java/isolated/ReloadableJava17JavadocVisitor.java

```diff
@@ -753,8 +756,16 @@ private JavaType.Variable fieldReferenceType(DCTree.DCReference ref, @Nullable J
 
     @Override
     public Tree visitReturn(ReturnTree node, List<Javadoc> body) {
-        body.addAll(sourceBefore("@return"));
-        return new Javadoc.Return(randomId(), Markers.EMPTY, convertMultiline(node.getDescription()));
+        List<Javadoc> before;
+        Markers markers = Markers.EMPTY;
+        if (source.startsWith("{", cursor)) {
+            markers = markers.addIfAbsent(Markers.build(Collections.singletonList(new LeadingBraceMarker(UUID.randomUUID()))));
```

Comment:
A few changes are necessary here:

- `addIfAbsent` adds a new Marker if it doesn't exist. So, the argument expects a `Marker`.
- Replace `UUID.randomUUID()` with `Tree.randomId()`

```suggestion
markers = markers.addIfAbsent(new LeadingBrace(Tree.randomId()));
```